### PR TITLE
Modified p4ovs_test_runner.py : skip any blank line, delay between ex…

### PR DESCRIPTION
**Pull request by dkaranjai**
…ecution of pre_tests and test script and between consecutive test script runs

Signed-off-by: Dwaipayan Karanjai <dwaipayan.karanjai@intel.com>

### CODE REVIEW

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-1/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/ef195d31af234a680e2e6f1a4e3be43b5a4f38bd

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-2/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/ef195d31af234a680e2e6f1a4e3be43b5a4f38bd

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-3/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/4ff83c885f4d2598b496fc27c2f61f3db9540e8e

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-4/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

**Comment by ipdksw** [comment]
Result:FAILURE JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-5/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-6/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-7/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

**Comment by dkaranjai** [comment]
Attaching the log from CI run:
[ptf_auto_consoleText.txt](https://github.com/ipdk-io/ovs/files/9415979/ptf_auto_consoleText.txt)

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-8/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

**Comment by ipdksw** [comment]
Result:FAILURE JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-9/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

File: ptf_tests/p4ovs_test_runner.py [hunk 1]
```diff
@@ -110,8 +115,10 @@ def print_help(self):
 
 results = {}
 for test in test_to_run['sequence']:
+    time.sleep(2)
     process = subprocess.Popen('/bin/bash', stdin=subprocess.PIPE, stdout=subprocess.PIPE)
     cmd = f"""source pre_test.sh {args.p4sde_install_path} {args.p4ovs_install_path} {args.p4dep_install_path}
+    sleep 2
```
Comment (5abeel) [inline ptf_tests/p4ovs_test_runner.py:161]: This should be `time.sleep(2)`

**Comment by 5abeel** [comment]
General comment: Recommend using a title and description breakup of commit message. Currently, we have the title showing a long sentence that breaks and rest added in the description. Please avoid this in the future PRs

**Review by 5abeel — CHANGES_REQUESTED** [review]

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-11/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-12/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-13/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-14/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6595d95dc47725b3601c314d422a9eaab5a25d39

File: ptf_tests/p4ovs_test_runner.py (hunk 1 above)
Comment (dkaranjai) [reply ptf_tests/p4ovs_test_runner.py:161]: this is in shell not python

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-55-15/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/821c2b3b5f00c9dc9632a820f043f73bc365666b

**Review by 5abeel — APPROVED** [review]
lgtm

**Comment by dkaranjai** [comment]
Can this be merged? We need one more approval for that.

**Review by nupuruttarwar — APPROVED** [review]
LGTM

**Pull request merged**